node-typescript/github-issue-bot/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+/* Throws an error if any of the keys are missing from the object. */
+
```

Comment:
Can we remove the space and use the docstring comment format?
e.g.

/*
 * Blah bla blah ...
 */